chapters/connectors.tex

```diff
@@ -119,6 +119,8 @@ \subsection{Expandable Connectors}\label{expandable-connectors}
   \item
     If the variable on the other side of the \lstinline!connect!-equation is \lstinline!input! or \lstinline!output! the new component will be either \lstinline!input! or \lstinline!output! to satisfy the restrictions in \cref{restrictions-of-connections-and-connectors} for a non-expandable connector.
     \begin{nonnormative}
+    The general rule ensures consistency for inside and outside connectors, and handles multiple connections to the new component.
+    For the simple case of no other connections involving these variables it can be explained as follows:
```

Comment:
Inside a paragraph (unlike before a listing or an item list) I find it odd to have a colon between sentences.
```suggestion
    For the simple case of no other connections involving these variables it can be explained as follows.
```